internal/conditions/conditions_test.go

```diff
@@ -17,11 +17,11 @@ func TestNewConditionForReason(t *testing.T) {
 		assert.Equal(t, "ReconcileSucceeded", condition.Reason)
 	})
 	t.Run("should create new condition for given predefined Reason with status False", func(t *testing.T) {
-		condition := ConditionFromExistingReason("OlderCRExists", "Other CR is elected a leader")
+		condition := ConditionFromExistingReason("WrongNamespaceOrName", "Only btpoperator kyma-system/btpoperator (namespace/name) is supported.")
```

Comment:
Same here: If it's not self-explanatory to the user, I'd add some information on what is required of them to handle/fix the error. This could be: "Use the correct namespace or name (format?) for your resource". Or, more straightforwardly, "Your resource must be in the kyma-system namespace. The resource's name must be btpoperator ."